consensus/src/event_processor_test.rs

```diff
@@ -151,13 +153,19 @@ impl NodeSetup {
         let network_events = network_events.map_err(Into::<anyhow::Error>::into);
         let author = signer.author();
 
-        // Bano
-        //playground.add_node(author, consensus_tx, network_reqs_rx, conn_mgr_reqs_rx);
-
+        // Bano: Using a dummy node ID
+        let peer_id = 1;
         let nodeId = NodeId {
-            id: 1,
-            author: author
+            id: peer_id,
+            author,
         };
+
+        // Update author_to_nodeIds which maps authors to node IDs
+        // (An author may have multiple node IDs for Twins)
+        playground
+            .author_to_nodeIds
+            .insert(author.clone(), vec![nodeId.clone()]);
```

Comment:
this should move into `add_node` function

Reply:
Addressed in similar comment below.